package.json

```diff
@@ -25,25 +25,22 @@
     "README.md",
     "LICENSE"
   ],
-  "resolutions": {
-    "minimist": "^1.2.2"
-  },
   "devDependencies": {
     "@mscs/eslint-config": "^0.1.2",
-    "@types/jest": "^25.1.4",
-    "@types/node": "^14.0.1",
-    "@typescript-eslint/eslint-plugin": "^2.24.0",
+    "@types/jest": "^26.0.0",
+    "@types/node": "^12.12.47",
```

Comment:
Donwgrade?

Reply:
We are currently support the version node@12, 14 is to high.